x-pack/plugins/security_solution/server/endpoint/services/actions/clients/crowdstrike/crowdstrike_actions_client.ts

```diff
@@ -321,6 +297,41 @@ export class CrowdstrikeActionsClient extends ResponseActionsClientImpl {
     return this.fetchActionDetails(actionRequestDoc.EndpointActions.action_id);
   }
 
+  async completeCrowdstrikeAction(
+    actionResponse: ActionTypeExecutorResult<CrowdstrikeBaseApiResponse> | undefined,
+    doc: LogsEndpointAction
+  ): Promise<void> {
+    const createBaseDocument = (): LogsEndpointActionResponse => ({
+      '@timestamp': moment().toISOString(),
+      agent: doc.agent,
+      EndpointActions: {
+        action_id: doc.EndpointActions.action_id,
+        completed_at: moment().toISOString(),
+        started_at: moment().toISOString(),
+        data: doc.EndpointActions.data,
```

Comment:
I wonder if we should add a suffix to the comment explaining how we complete it.